cap-ci/pipeline.template

```diff
@@ -67,6 +69,46 @@ resources:
 
 {{ $pipelineName := .Env.PIPELINE }}
 
+{{- $create_k8s := `
+            # create k8s cluster
+            export random_variable=$(hexdump -n 8 -e '2/4 "%08x"' /dev/urandom)
+            if [[ ${BACKEND} == "gke" ]]; then
+              printf "%s" '((gke-key-json))' > $PWD/gke-key.json
+              export GKE_CRED_JSON=$PWD/gke-key.json
+              export GKE_CLUSTER_NAME=ci-${BACKEND}-${random_variable}
+              export CLUSTER_NAME=${GKE_CLUSTER_NAME}
+              (cd catapult; make BACKEND=$BACKEND GKE_CREDS_JSON=$GKE_CRED_JSON GKE_CLUSTER_NAME=${GKE_CLUSTER_NAME} CLUSTER_NAME=${GKE_CLUSTER_NAME})
+              echo ${GKE_CLUSTER_NAME} > kubeconfig-pool/name
+              cp catapult/build${GKE_CLUSTER_NAME}/kubeconfig kubeconfig-pool/metadata
+            else
+              echo "CI cannot create ${BACKEND} cluster"
+              exit 1
+            fi
+` }}
+
+{{- $delete_k8s := `
+                  # delete k8s cluster
+                  if [[ ${BACKEND} == "gke" ]]; then
+                      echo "ls"
+                      ls
+                      pwd
+                      which gcloud
+                      printf "%s" '((gke-key-json))' > $PWD/gke-key.json
+                      export GKE_CRED_JSON=$PWD/gke-key.json
+                      export CLUSTER_NAME=$(cat pool.kube-hosts/name)
+                      pool_file=${pool_file:-pool.kube-hosts/metadata}
+                      export GKE_CLUSTER_ZONE=$(cat ${pool_file} | grep current-context | cut -d'_' -f3)
+                      gcloud auth activate-service-account --key-file=${GKE_CRED_JSON}
+                      PROJECT=$(jq -r .project_id gke-key.json)
```

Comment:
it is just using what $import_gke was already using. Plus this CI is private

Reply:
Is private to all people on SUSE, and outside contributors, which don't need access to our GKE credentials.

Implementing this the correct way would not leak private credentials.

Reply:
I am not sure how it might be leaking. There is already a key (implementation from `$import_gke `) . All I am doing is putting some part of it in a variable on the fly. We will need to interact with key to generate kubeconfigs 
https://github.com/SUSE/catapult/issues/175

Reply:
You are right, sorry. For a moment I thought the contents of the gke-key.json will be leaked. Resolving.